r/adbcdrivermanager/R/adbc.R

```diff
@@ -151,6 +160,11 @@ adbc_connection_set_options <- function(connection, options) {
 #' @rdname adbc_connection_init
 #' @export
 adbc_connection_release <- function(connection) {
+  if (isTRUE(connection$.release_database)) {
+    database <- connection$database
+    on.exit(adbc_database_release(database))
```

Comment:
Do you mean to release the database even in case of an error? What error can happen when releasing?